notifier/webhook/deliverer.go

```diff
@@ -83,12 +83,12 @@ func (d *Deliverer) Deliver(ctx context.Context, nID uuid.UUID) error {
 		Str("notification_id", nID.String()).
 		Logger()
 
-	callback := d.conf.callback
+	callback := *d.conf.callback
 	callback.Path = path.Join(callback.Path, nID.String())
```

Comment:
This is probably better, but does require the config to end with a slash:
```suggestion
	callback, err := d.conf.callback.Parse(nID.String())
	if err != nil {
		return err
	}
```